index-scheduler/src/batch.rs

```diff
@@ -416,41 +412,46 @@ impl IndexScheduler {
             )));
         }
 
-        // 5. We take the next task and try to batch all the tasks associated with this index.
-        if let Some(task_id) = enqueued.min() {
-            let task = self.get_task(rtxn, task_id)?.ok_or(Error::CorruptedTaskQueue)?;
+        // 5. We make a batch from the unprioritised tasks. Start by taking the next enqueued task.
+        let task_id = if let Some(task_id) = enqueued.min() { task_id } else { return Ok(None) };
```

Comment:
Does it pass rustfmt, here? 😂 I love that but not sure it will pass!

Reply:
I wrote it on several lines and rustfmt put it all in one line haha >:-(